Tests/SecureXPCTests/Client & Server/Round-trip Integration Test.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  Round-trip Integration Test.swift
+//  
+//
+//  Created by Alexander Momchilov on 2021-11-28.
+//
+
+import XCTest
+import SecureXPC
+
+class RoundTripIntegrationTest: XCTestCase {
+    var xpcClient: XPCClient! = nil
+
+
+    let anonymousServer = XPCServer.newAnonymousService()
```

Comment:
So I understand this correctly all tests in this test case reuse the same server, but a new client is created each time. Wouldn't a new server for each test case so routes don't persist between test cases?

Reply:
Nope, the service is new, too.

[XUnit testing frameworks](https://en.wikipedia.org/wiki/XUnit) create one instance of the test class per test being run. This is done specifically to avoid test state being shared by instance variables.

Reply:
Heh, did you just upvote https://stackoverflow.com/a/68655018/3141234 ?

Reply:
> XUnit testing frameworks create one instance of the test class per test being run. This is done specifically to avoid test state being shared by instance variables.

Ah, I didn't realize that. Good to know!

> Heh, did you just upvote https://stackoverflow.com/a/68655018/3141234 ?

Never seen that post before (I don't typically read `ios` tagged posts), but it's helpful to read! Considering how this works, a bit unclear to me why `setUp()` exists instead of `init()` or similar. But this chain of thought goes well beyond this PR.

Reply:
> Never seen that post before

Ah, I saw this upvoted recently, and it's not a popular topic/question/answer, so I figured it might have been you, haha.

> a bit unclear to me why `setUp()` exists instead of `init()` or similar.

Few thoughts:

1. There's two initializers, so it might not be clear which one to override. (Not sure if one is just an "convenience" initializer, though).
2. Objective-C didn't have definite-variable-initialization, so there was no problem with leaving variables uninitialized in the init, and delegating that to a function like `setUp()`
3. The initializers use some pretty behind-the-scenes concepts, that perhaps Apple didn't want to expose into every single consumer.